aiida-tutorials/structures/qmof-00d09fe.cif

```diff
@@ -0,0 +1,48 @@
+# generated using pymatgen
```

Comment:
Ideally these files would probably go into `data`, unless there's a reason to keep them separate? Maybe not worth fixing before tomorrow though